src/lib/utils/sendBeacon.spec.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ *
+ * stylelounge.de
+ *
+ * Copyright (C) SNM Style Net Media GmbH
+ * MIT Licensed
+ */
+import {default as send} from "./sendBeacon";
+import * as sinon from "sinon";
+import IManifest from "../types/IManifest";
+declare let global: any;
+describe("sendBeacon ", () => {
+    let manifest: IManifest;
+    before(() => {
+        manifest = {
+            data: {
+                mock: "data",
+            },
+            id: 1,
+            url: "/some-mock-url",
+            verb: "POST",
+        };
+    });
+    beforeEach(() => {
```

Comment:
let's always use `function` instead of `() =>` for `describe` and `it`, there is occasional weird behavior with plugins for chai/mocha so we agreed to always use `function`